src/app/min/kmong/(header)/page.tsx

```diff
@@ -0,0 +1,182 @@
+"use client"
+
+import Image from "next/image"
+import Link from "next/link"
+import { useEffect, useState } from "react"
+
+const list = ["react", "html", "javascript", "next", "css"]
+
+const list2 = [
+  "홈페이지제작",
+  "홈페이지",
+  "워드프레스",
+  "크롤링",
+  "카페24",
+  "블로그",
+  "아임웹",
+  "카페24",
+  "블로그",
+  "아임웹",
+]
+
+export default function Header() {
+  const [searchState, toggleSearchState] = useState(false)
+  const [isSticky, setIsSticky] = useState(false)
+
+  useEffect(() => {
+    window.addEventListener("scroll", handleScroll)
+    return () => {
+      window.removeEventListener("scroll", handleScroll)
+    }
+  }, [])
+
+  const handleScroll = () => {
+    const threshold = window.innerHeight
+    setIsSticky(window.scrollY > threshold)
+  }
+
+  const handleFocus = () => {
+    toggleSearchState(true)
+  }
+
+  const handleBlur = () => {
+    toggleSearchState(false)
+  }
+
+  return (
+    <>
+      <div className="w-full bg-[#212224] h-13 text-center my-auto">
+        <Link
+          href={"/"}
+          className="w-full block leading-[3rem] text-white text-lg font-normal">
+          신규 가입하면
+          <span className="text-[#f5c126] font-semibold"> 10만원 쿠폰 </span>
+          드려요 〉
+        </Link>
+      </div>
+      <header
+        className={`flex-col sticky top-0 ${
+          isSticky ? "border-b border-gray-300" : ""
+        } `}>
+        <div className="flex items-center justify-center h-16 w-full	 bg-white">
+          <div className="mr-6">
+            <Link href={"/"}>
+              <Image src="/kmong_logo.gif" alt="로고" width={100} height={52} />
+            </Link>
+          </div>
+          <div className="mx-4 relative">
+            <input
+              type="search"
+              className="border border-black rounded-3xl h-11 w-[600px] outline-none hover:bg-gray-200 px-4 placeholder:text-black placeholder:text-sm focus:placeholder:text-opacity-0"
+              placeholder="내년 운세가 궁금하다면? 답답한 순간 속 시원한 상담"
+              onFocus={handleFocus}
+              onBlur={handleBlur}
+            />
+            <button>
+              <SearchIcon />
+            </button>
+            <InputDialog searchState={searchState} />
+          </div>
+          <div className="flex">
+            <ul className="flex gap-5 items-center text-sm">
+              <li className="flex gap-1">
+                <span className="items-center leading-5">엔터프라이즈</span>
+                <span className="text-blue-500 bg-blue-100 font-bold rounded-xl px-1.5 text-[10px]">
+                  기업용
+                </span>
+              </li>
+              <li>
+                <Link href={"/"}>전문가 등록</Link>
+              </li>
+              <li>
+                <Link href={"/"}>로그인</Link>
+              </li>
+              <li>
+                <button className="bg-[#f5c126] rounded-lg py-1.5 px-3 font-medium hover:bg-yellow-500 active:bg-yellow-600 ml-2">
+                  회원가입
+                </button>
```

Comment:
페이지 이동은 추후에 `onClick{() => router.push('/signup')}` 으로 설정하실 건가요?

--- 
가독성 위해 tailwind 속성값 순서 변경 추천

```suggestion
                <button className="bg-[#f5c126] rounded-lg py-1.5 px-3 font-medium ml-2 hover:bg-yellow-500 active:bg-yellow-600">
                  회원가입
                </button>
```

Reply:
아직 이동되는 사이트가 없어 따로 설정안했습니다.
추후에 적용하겠습니다.